src-tauri/Cargo.toml

```diff
@@ -18,6 +18,7 @@ tauri-build = { version = "1.5.3", features = [] }
 serde_json = "1.0.109"
 serde = { version = "1.0.193", features = ["derive"] }
 tauri = { version = "1.7.1", features = ["api-all", "devtools", "updater"] }
+tauri-plugin-window-state = { git = "https://github.com/tauri-apps/plugins-workspace", branch = "v1" }
```

Comment:
Using git url will break the flatpak builds, please use it from crates io.

Reply:
Thx! Let me know if anything else comes up 🤍